package.json

```
@@ -44,27 +44,27 @@
     "prepublishOnly": "npm run build"
   },
   "devDependencies": {
-    "@capacitor/android": "^4.6.1",
-    "@capacitor/cli": "^4.6.1",
-    "@capacitor/core": "^4.6.1",
-    "@capacitor/docgen": "^0.2.0",
-    "@capacitor/ios": "^4.6.1",
+    "@capacitor/android": "^4.7.0",
+    "@capacitor/cli": "^4.7.0",
+    "@capacitor/core": "^4.7.0",
+    "@capacitor/docgen": "^0.2.1",
+    "@capacitor/ios": "^4.7.0",
     "@ionic/eslint-config": "^0.3.0",
     "@ionic/swiftlint-config": "^1.1.2",
     "@mobilejazz/prettier-config": "^0.1.2",
-    "eslint": "^8.32.0",
-    "eslint-plugin-import": "^2.27.4",
-    "prettier": "~2.8.3",
+    "eslint": "^8.35.0",
+    "eslint-plugin-import": "^2.27.5",
+    "prettier": "~2.8.4",
     "prettier-plugin-java": "~2.0.0",
-    "rimraf": "^4.0.7",
-    "rollup": "^3.10.0",
+    "rimraf": "^4.4.0",
+    "rollup": "^3.18.0",
     "swiftlint": "^1.0.1",
-    "typescript": "~4.9.4"
+    "typescript": "~4.9.5"
   },
   "peerDependencies": {
-    "@capacitor/core": "^4.6.1",
-    "@bugfender/sdk": "^2.2.1",
-    "@bugfender/common": "^1.1.2"
+    "@capacitor/core": "^4.7.0",
```

This maybe is better:
```suggestion
    "@capacitor/core": "^4.0.0",
```